.travis.yml

```diff
@@ -1,16 +1,17 @@
 sudo: false
 rvm:
+  - '2.5'
   - '2.4'
   - '2.3'
 
 notifications:
-  email:
-    - tim@github.com
+  email: false
 env:
   - PUPPET_VERSION="~> 5.3.0"
   - PUPPET_VERSION="~> 5.2.0"
   - PUPPET_VERSION="~> 5.1.0"
   - PUPPET_VERSION="~> 5.0.0"
+  - PUPPET_VERSION="~> 6.0"
```

Comment:
I also meant that here the newest should be on top. Probably not too important, but I'm a big fan of teaching yourself the right way so you don't need to think about it next time :)